test/UKHO.ADDS.EFS.Builder.S100.API.FunctionalTests/Helpers/FileHelpers.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.IO.Compression;
+
+namespace UKHO.ADDS.EFS.Orchestrator.API.FunctionalTests.Helpers
```

Comment:
Extracting large zip files to disk may be slow and consume disk space please check if we can improve it.
```suggestion
namespace UKHO.ADDS.EFS.Builder.S100.API.FunctionalTests.Helpers
```